reference/clinic.v1.yaml

```diff
@@ -1273,6 +1273,132 @@ paths:
       tags:
         - Clinics
         - Internal
+  /v1/redox:
+    post:
+      summary: Redox EHR Endpoint
+      operationId: ProcessEHRMessage
+      responses:
+        '200':
+          description: OK
+      tags:
+        - Redox
+      description: An endpoint for incoming Redox messages
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
```

Comment:
Is the plan for this to reference the `NewOrder` model from redox.v1.yaml?

Reply:
No - we've had issues with the API validation library so I'm going to assume whatever message we receive are valid (thus having only an object here). Otherwise we risk blocking the queue processing.